plugin/terway/cni_linux.go

```diff
@@ -374,9 +374,16 @@ func doCmdDel(ctx context.Context, logger *logrus.Entry, client rpc.TerwayBacken
 						if err != nil {
 							return err
 						}
-						break
+						continue
 					}
 				}
+				fallthrough
```

Comment:
If `IPvlan` not support, the `case types.PolicyRoute:` condition also not satisfy